.lefthook.yml

```diff
@@ -0,0 +1,13 @@
+assert_lefthook_installed: true
+output: [execution, failure, summary]
+pre-commit:
+  parallel: true
+  commands:
+    fmt:
+      run: pnpm dprint fmt
+      stage_fixed: true
```

Comment:
this means formatted files will be staged for the commit the hook is running for